pkgs/development/python-modules/hickle/default.nix

```diff
@@ -19,31 +18,23 @@
 
 buildPythonPackage rec {
   pname   = "hickle";
-  version = "4.0.4";
+  version = "5.0.2";
   disabled = pythonOlder "3.5";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0d35030a76fe1c7fa6480088cde932689960ed354a2539ffaf5f3c90c578c06f";
+    hash = "sha256-2+7OF/a89jK/zLhbk/Q2A+zsKnfRbq3YMKGycEWsLEQ=";
   };
 
-  postPatch = ''
-    substituteInPlace requirements_test.txt \
-      --replace 'astropy<3.1;' 'astropy;' --replace 'astropy<3.0;' 'astropy;'
-  '';
-
   propagatedBuildInputs = [ h5py numpy dill ];
 
-  doCheck = false; # incompatible with latest astropy
   checkInputs = [
-    pytest pytest-cov pytest-runner coveralls scipy pandas astropy twine check-manifest codecov
+    pytestCheckHook pytest-cov coveralls scipy pandas astropy twine check-manifest codecov
```

Comment:
I got the feedback that we don't want to do coverage tests. Maybe we should remove `pytest-cov` and `codecov` by patching or disabling tests

Reply:
@SuperSandro2000

Reply:
Patched out.